lisp/plugins/gst_backend/gst_backend.py

```diff
@@ -120,7 +121,21 @@ def _add_uri_audio_cue(self):
         if files:
             GstBackend.Config["mediaLookupDir"] = os.path.dirname(files[0])
             GstBackend.Config.write()
-
+        self._cue_audio_files(files)
+
+    def cue_audio_urls(self, urls):
```

Comment:
I would name this `add_cue_from_urls`, it makes (at least to me) the use of the method more clear.